kbase-extension/scss/partials/_narrative.scss

```diff
@@ -2837,7 +2835,7 @@ button.kb-data-obj {
   width: calc(100% - 6px);
   height: 80%;
   justify-content: center;
-  padding: 0;
+  padding: 10px 0;
```

Comment:
this fixes the 'Copy' button in the 'My Data' window, which I think had been affected by css changes elsewhere, as the text was not centred in the container.